### ISSUE TITLE
Enable targeted EAPOL PMKID Scan on Marauder 6.1 standalone UI

### ISSUE THREAD

**Issue by ndguardian** [justcallmekoko/ESP32Marauder#443]
**Is your feature request related to a problem? Please describe.**
Full disclosure, I first started out with ESP32Marauder on the Flipper Zero. It's possible that this feature does exist on the standalone versions and I'm just being dumb.

On the Flipper Zero, there is an option to select the APs you want to target from the list of scanned APs, and from there you can do a PMKID sniff against that list exclusively from the UI. From reading the docs, it sounds like the core functionality exists on the standalone units, but doing so requires CLI access exclusively rather than via the UI. Via the UI, scanning APs, selecting APs for targeted attacks and performing PMKID sniffs all exist, but it doesn't seem you can do that workflow all together via the touchscreen UI.

This is problematic as I live in an apartment complex with quite a few WiFi networks around, and if I want to target exclusively my own WiFi for this I can't do that standalone. It picks up the WiFi present on whatever channel I happen to be on instead, meaning it also captures my neighbors' WiFi.

**Describe the solution you'd like**
Basically just enable the workflow as described above. Scan APs > Select APs > Sniff PMKID/EAPOL on selected APs

**Describe alternatives you've considered**
Obviously I can stick with the Flipper Zero, but this obviously is unhelpful for anyone else who doesn't have one. Could also use the CLI, but that doesn't work as a purely standalone solution either.

**Additional context**
No other context, but just want to also say excellent work on this project so far!
